### PR TITLE
Created welcome page to navigate between CRUDs for different entities.

### PR DESCRIPTION
**Pull request by BrentonLenzen**

### CODE REVIEW

**Comment by jNLeee** [comment]
No problems when running and looks ready to merge!

**Comment by anujparakh** [comment]
Looks good! Just one small question - there's a semester.scss and another semesters.scss stylesheet. Don't know if that's on purpose or not.

File: AFISO-Tracker/app/controllers/semesters_controller.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	end
+  end
+
+  def edit
+	@semester = Semester.find(params[:id])
+  end
+  
+  def update
+	@semester = Semester.find(params[:id])
+	puts semester_params[:published_date]
```
Comment (Amg9973) [inline AFISO-Tracker/app/controllers/semesters_controller.rb:31]: This puts method can probably be removed now

Comment (Amg9973) [reply AFISO-Tracker/app/controllers/semesters_controller.rb:31]: Other than that it everything looks good and ready to merge.

**Pull request merged**